draft-todo-chariton-dns-account-01.mkd

```diff
@@ -82,22 +85,19 @@ When the identifier being validated is a domain name, the client can prove contr
 
 A client can fulfill this challenge by performing the following steps:
 
-Construct a key authorization from the "token" value provided in the challenge and the client's account key \\
-Compute the SHA-256 digest {{FIPS180-4}} of the key authorization \\
-Construct the validation domain name by prepending the following label to the domain name being validated:
+- Construct a key authorization from the "token" value provided in the challenge and the client's account key
+- Compute the SHA-256 digest {{FIPS180-4}} of the key authorization
+- Construct the validation domain name by prepending the following label to the domain name being validated:
 
-~~~
-"_acme-challenge_" || base32(SHA-256(Account Resource URL)[0:10))
-~~~
-
-SHA-256 is the SHA hashing operation defined in {{!RFC6234}}. \\
-[0:10) is the operation that selects the first ten bytes from the previous SHA256 operation. \\
-base32 is the operation defined in {{!RFC4648}} \\
-Account Resource URL is defined in {{!RFC8555}}
+      "_acme-challenge_" || base32(SHA-256(Account Resource URL)[0:10))
```

Comment:
No ~~~ here?

Reply:
No since it's indented its considered a codeblock.

Reply:
But why not have all codeblocks use `~~~` and have all of them except one like that? Does it help somehow with the indentation?

Reply:
I was having issues with not doing this, I'll give it a bit more try.

Reply:
Yep kramdown seems...difficult to deal with. I didn't find a good way.